ecodan-ha-local/ehal_hp.cpp

```diff
@@ -385,6 +385,25 @@ namespace ehal::hp
         return true;
     }
 
+    bool set_power_mode(bool on)
+    {
+        Message cmd{MsgType::SET_CMD, SetType::BASIC_SETTINGS};
+        cmd[1] = SET_SETTINGS_FLAG_SYSTEM_MODE_POWER;
+        cmd[3] = on ? 1 : 0;
+        {
+            std::lock_guard<std::mutex>{cmdQueueMutex};
```

Comment:
This lock guard object is creating a temporary which is being immediately destroyed because there is no variable name, and it's not actually locking anything in the expected scope. I guess it's copy pasted from somewhere else I already made that mistake! Should be:

`std::lock_guard<std::mutex> lock{cmdQueueMutex};`

Reply:
Hi, sorry. I'm java developer and not used to C++ memory management :). It was copy/pasted. I correct.